.github/workflows/tests-swizzle.yml

```diff
@@ -0,0 +1,31 @@
+name: Swizzle Tests
```

Comment:
FYI @Josh-Cena this is how I plan to test the swizzle CLI

Ensure that our site can build with swizzle/wrap with js/ts + typecheck for all theme-classic components

It's not perfect, no test to see it works visually, but better than nothing